phoenicis-repository/src/main/java/org/phoenicis/repository/dto/RepositoryDTO.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2015-2017 PÂRIS Quentin
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package org.phoenicis.repository.dto;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import java.util.*;
+
+/**
+ * Represents a repository
+ */
+@JsonDeserialize(builder = RepositoryDTO.Builder.class)
+public class RepositoryDTO {
+    private final String name;
+    private final List<CategoryDTO> categories;
+    private final Map<String, String> translations;
```

Comment:
In general, we avoid using Map<String, String> in DTOs at it does not really say something about the data held.

Could we use a Set of TranslationDTO instead